tools/ZeroC.Ice.Slice.Tools.CSharp/README.md

```diff
@@ -0,0 +1,20 @@
+# Slice Tools for Ice for C#
+
+ZeroC.Ice.Slice.Tools.CSharp allows you to compile Slice definitions (in `.ice` files) into C# code (in `.cs` files)
+within MSBuild projects.
+
+This package includes the Slice compiler for C#, `slice2cs`. This compiler is a native tool with binaries for
+Linux (x64 and arm64), macOS (x64 and arm64) and Windows (x64).
+
+Once you've added the ZeroC.Ice.Slice.Tools.CSharp NuGet package to your project, the Slice files of your project are
+automatically compiled into C# files every time you build this project.
+
+> The Slice MSBuild task checks the dependencies of the Slice files it compiles and only rebuilds the Slice files that,
+> are not up to date.
+
+[Source code][source] | [Package][package] | [slice2cs options][slice2cs] | [Slice documentation][slice]
+
```

Comment:
Do you plan to add more information in a follow-up PR?

See https://github.com/icerpc/icerpc-csharp/tree/main/tools/IceRpc.Slice.Tools